src/cli/task-generate.ts

```diff
@@ -40,15 +40,17 @@ export const taskGenerate = async (coreCompiler: CoreCompiler, config: Validated
     config.flags.unknownArgs.find((arg) => !arg.startsWith('-')) ||
     ((await prompt({ name: 'tagName', type: 'text', message: 'Component tag name (dash-case):' })).tagName as string);
 
+  if (undefined === input) return;
```

Comment:
Can you wrap this conditional in braces please? It would also be helpful to explain what we're attempting to resolve:
```suggestion
  if (undefined === input) {
    // in some shells (e.g. Windows PowerShell), hitting Ctrl+C results in a TypeError printed to the console.
    // explicitly return here to avoid printing the error message.
    return;
  }
```